docs/samples/pipelines/sample-tf-pipeline.py

```diff
@@ -27,10 +26,14 @@ def kfservingPipeline(
     action = 'create',
     model_name='tensorflow-sample',
     default_model_uri='gs://kfserving-samples/models/tensorflow/flowers',
-    canary_model_uri='gs://kfserving-samples/models/tensorflow/flowers-2',
+    canary_model_uri='gs://kfserving-samples/models/tensorflow/flowers',
     canary_model_traffic_percentage='10',
     namespace='kubeflow',
-    framework='tensorflow'
+    framework='tensorflow',
+    default_custom_model_spec='{}',
+    canary_custom_model_spec='{}',
+    autoscaling_target=0,
+    kfserving_deployer_api=''
```

Comment:
what's `kfserving_deployer_api` ?

Reply:
changed it kfserving_endpoint. This is for scenarios when kfserving is running remote.